docs/guide/mcp-servers.md

```
@@ -0,0 +1,116 @@
+
+## Example: Filesystem Server
+
+The MCP project provides a reference filesystem server. To set it up:
+
+**Server name**: `filesystem`
+
+**Command**: `npx`
+
+**Arguments** (one per line):
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)